src/app/lib/utilities/preprocessor/rules/addCPSHeadingBlock.js

```diff
@@ -0,0 +1,28 @@
+import { clone, path } from 'ramda';
+
+const buildHeadingBlock = text => ({
+  text,
+  markupType: 'plain_text',
+  type: 'heading',
+});
+
+export const headingResolver = headlines =>
+  headlines.headline.replace(/<[^>]*>/g, '');
+
+export default payload => {
+  const output = clone(payload);
+
+  const headlines = path(['promo', 'headlines'], output);
+  const blocks = path(['content', 'blocks'], output);
+
+  if (!headlines || !blocks) {
+    return output;
```

Comment:
Do we have tests that cover both passing no `headlines` and no `blocks` to check the `output` is returned?